CMakePresets.json

```diff
@@ -0,0 +1,172 @@
+{
+    "version": 3,
+    "cmakeMinimumRequired": {
+        "major": 3,
+        "minor": 19,
+        "patch": 0
+    },
+    "configurePresets": [
+        {
+            "name": "debug-flags",
+            "hidden": true,
+            "cacheVariables": {
+                "CMAKE_BUILD_TYPE": "Debug",
+                "CMAKE_C_FLAGS": "-fno-omit-frame-pointer -fno-optimize-sibling-calls -gfull -O0",
+                "CMAKE_CXX_FLAGS": "-fno-omit-frame-pointer -fno-optimize-sibling-calls -gfull -O0"
+            },
+            "environment": {
+                "VCPKG_SUFFIX": ""
```

Comment:
This means we would use the host system's default vcpkg triplet, which cxx-common does not distribute. Is that your intention?

Technically, we are still be able to build remill with debug symbols without requiring everything else to have debug symbols.

Reply:
I think I am missing something here and need a bit more of an explanation about the issue.

Reply:
cxx-common only builds the `x64-{linux,osx}-rel` triplet (value for `VCPKG_TARGET_TRIPLET`), so if a user uses this preset, they would need to have built cxx-common themselves in order to use the `x64-{linux,osx}` triplet that the empty `VCPKG_SUFFIX` will generate.

I worry that people will get very confused when vcpkg spits out an error that it can't find things when trying to build debug versions of remill.

Reply:
Since we now take care of the triplet in the script, should we get rid of the `VCPKG_SUFFIX` in the presets file?

Reply:
Yes 🙏